package.json

```diff
@@ -52,6 +54,7 @@
   "dependencies": {
     "debug": "^4.3.4",
     "lodash": "^4.17.21",
+    "node-insim": "^0.9.1",
```

Comment:
Node InSim depending on itself now? 😄

Reply:
pitfalls of using "too smart" IDE. fixed.